tests/test_equilibrium.py

```diff
@@ -201,6 +201,31 @@ def test_resolution():
     assert eq1.R_basis.NFP == 5
 
 
+@pytest.mark.unit
+def test_symmetry():
+    """Test changing equilibrium symmetry."""
+    eq_sym1 = Equilibrium(M=3, N=2, sym=True)
```

Comment:
the default surface is symmetric and pretty simple, might be a better test to use a more complicated geometry to make sure things are truncated correctly